OpenRA.Mods.Cnc/Util.cs

```diff
@@ -13,28 +13,42 @@ namespace OpenRA.Mods.Cnc
 {
 	public static class Util
 	{
+		// TD and RA used a nonlinear mapping between artwork frames and unit facings.
+		// These tables define the facing ranges used for units with 16 and 32 rotation images.
+		// i.e. image 1 is used for facings 8-23, image 2 for 24-40, and so on.
+		// Image 0 is used for facings smaller than the first entry or equal or larger than the last entry.
+		static readonly int[] Facing16 =
+		{
+			8, 24, 40, 56, 72, 88, 104, 120,
+			136, 152, 168, 184, 200, 216, 232, 248
```

Comment:
This seems to just add +16 at every step. Can you explain why it needs a lookup table?

Reply:
Hmm, you appear to be right - a derp on my part. I can remove this and the second commit!